### PR TITLE
implement initial for Tutorial pane

### PR DESCRIPTION
**Pull request by kevinushey**
Brings the Tutorial tab to life, albeit currently with no content. Only displayed in devmode for now.

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml [hunk 1]
```diff
@@ -1377,6 +1379,12 @@ well as menu structures (for main menu and popup menus).
         menuLabel="Zoom VCS"
         windowMode="main"/>
         
+   <cmd id="layoutZoomTutorial"
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml]: Also think about having a command on the View menu for moving focus to the Tutorial pane. Important for accessibility, both screen reader users and for keyboard-only users.

This area is a bit of a mess, see #5775 for more considerations (nothing you need to do about it at the moment!)

**Comment by coatless** [comment]
Any chance we could get a screenshot to show what the "tutorial mode" looks like c.f. #3018?

**Comment by jmcphers** [comment]
Nothing to see yet, but this will be done by the time RStudio 1.3 is released.

**Comment by coatless** [comment]
@jmcphers do you folks have a storyboard or any concept work? I'm just curious to see how it will be implemented to plan out a few teaching approaches. Also, the release target for RStudio v1.3 is likely Spring 2020 post-R 4.0.0?

**Comment by jmcphers** [comment]
Nothing we can share, sorry. But the broad strokes are that there will be a Tutorial pane which can host learnr tutorials. Those tutorials will run in a background R process, which will communicate back to the main R session using rstudioapi calls so that they can control the IDE without blocking it. We will be adding a few new rstudioapi methods to make building tutorials with learnr easier; e.g. an API to highlight or execute a particular command. 

https://rstudio.github.io/learnr/

**Pull request merged**